Sources/PublicInterface/Server.swift

```diff
@@ -26,6 +26,9 @@ public protocol ServerDelegate: AnyObject {
 
     /// Called only when the session is updated with intention of the dAppt.
     func server(_ server: Server, didUpdate session: Session)
+    
+    /// Called when not able to parse a message
+    func server(_ server: Server, couldNotParseMessage message: String, from url: WCURL, error: Error)
```

Comment:
Could you please move this method to `ServerDelegateV2` protocol? This way, this doesn't break existing code

Reply:
if you implemented `ServerDelegateV2` you will run into the same issue, should I create a `ServerDelegateV3`?

Reply:
@DmitryBespalov this is probably going to be unmaintained soon, do you still want this PR to be merged?